chore: add different templates for issue (#1853)
DevExpress/devextreme-reactive: .github/ISSUE_TEMPLATE/bug_report.md

<!--
    We cannot guarantee a quick answer on GitHub.

>     If you own an active DevExtreme license, feel free to contact us in the Support Center (https://www.devexpress.com/ask)

Reviewer: ```suggestion
    If you have an active DevExtreme license, you can contact us in the Support Center (https://www.devexpress.com/ask)
```

Reviewer: ```suggestion
    If you have an active DevExtreme license, contact us in the Support Center (https://www.devexpress.com/ask)
```

>     to receive an answer shortly.

Reviewer: for a quicker response? a prompt reply?

Reviewer 2: ```suggestion
    for a quicker response.
```

Reviewer: Ok

-->

<!--
    A checked check box should look like this: [x]
-->

> - [ ] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

Reviewer: and believe?

Reviewer 2: > [believe](https://www.macmillandictionary.com/dictionary/british/believe#believe__1)
[transitive] to think that a fact is true

Reviewer: I know what believe means, however, 'believe' is a matter of perspective or opinion.

Reviewer 2: That is correct. The person that opens an issue is of opinion that he or she does not create a duplicate. However, it might be not true.

> - [ ] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

Reviewer: ```suggestion
- [ ] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) for duplicates.
```

Reviewer 2: I read it as "I've looked through all issues to find if any of them appears more than once". What we want the creator of an issue to do is to search the issues to find if there is already one that reports the problem the creator has.

Reviewer: ```suggestion
- [ ] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and did not find duplicates.
```

Reviewer 2: I'm afraid this version has the same issue.

### I'm using ...

- [ ] React Grid
- [ ] React Chart
- [ ] React Scheduler

### Current Behaviour

> <!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->

Reviewer: ```suggestion
<!-- A detailed description of the problem. For example, I'm always frustrated when [...] -->
```

Reviewer: Why 'I'm always frustrated when'?

Reviewer 2: That's just an example of how you can start your description of the problem
```suggestion
<!-- A detailed description of the problem. For example, "I'm always frustrated when [...]" -->
```

Reviewer: Ok

### Expected Behaviour

> <!-- A clear and concise description of what you want to happen. -->

Reviewer: ```suggestion
<!-- A detailed description of the expected result. -->
```

### Steps to Reproduce
<!--

>     Provide detailed steps to reproduce the bug. If possible, create a demo illustrating the problem.

Reviewer: ```suggestion
    Provide steps to reproduce the bug. If possible, create a demo illustrating the problem.
```

-->

### Screenshots

> <!-- Illustrate the problem with a couple screenshots if possible. -->

Reviewer: ```suggestion
<!-- Use screenshots to Illustrate the problem. -->
```

Reviewer 2: ```suggestion
<!-- Use screenshots to illustrate the problem if possible. -->
```
